### ISSUE TITLE
No matching version found for eslint@^7.74.0 when launch   vue create my-project --preset assurance-maladie-digital/vue-cli-preset

### ISSUE THREAD

**Issue by asicfr** [assurance-maladie-digital/vue-cli-preset#4]
Hello, I encountered a problem when launching the create project command on windows.
I use :
```
npm -v : 6.14.4
node -v : v12.18.0
```

Here is the console log :
```
vue create my-project --preset assurance-maladie-digital/vue-cli-preset
@cnamts/vue-cli-plugin-vue-dash
? Description du projet My kickass project
? Utiliser Vue i18n (pour l'internationalisation) No
? Utiliser Vuex Persist No
? Utiliser cypress pour les tests d'interface No
? Installer Matomo (pour les analyses d'audience) No
? Installer le FormBuilder No
 WARN  conflicting versions for project dependency "@types/jest":

- ^24.0.19 injected by generator "@vue/cli-plugin-unit-jest"
- ^26.0.23 injected by generator "@cnamts/vue-cli-plugin-vue-dash"

Using newer version (^26.0.23), but this may cause build errors.
 WARN  conflicting versions for project dependency "@vue/eslint-config-standard":

- ^5.1.2 injected by generator "@vue/cli-plugin-eslint"
- ^6.0.0 injected by generator "@cnamts/vue-cli-plugin-vue-dash"

Using newer version (^6.0.0), but this may cause build errors.
 WARN  conflicting versions for project dependency "eslint":

- ^6.7.2 injected by generator "@vue/cli-plugin-eslint"
- ^7.74.0 injected by generator "@cnamts/vue-cli-plugin-vue-dash"

Using newer version (^7.74.0), but this may cause build errors.
 WARN  conflicting versions for project dependency "eslint-plugin-promise":

- ^4.2.1 injected by generator "@vue/cli-plugin-eslint"
- ^5.1.0 injected by generator "@cnamts/vue-cli-plugin-vue-dash"

Using newer version (^5.1.0), but this may cause build errors.
 WARN  conflicting versions for project dependency "eslint-plugin-vue":

- ^6.2.2 injected by generator "@vue/cli-plugin-eslint"
- ^7.9.0 injected by generator "@cnamts/vue-cli-plugin-vue-dash"

Using newer version (^7.9.0), but this may cause build errors.
 WARN  conflicting versions for project dependency "sass-loader":

- ^8.0.2 injected by generator "@vue/cli-service"
- ^10.1.1 injected by generator "@cnamts/vue-cli-plugin-vue-dash"

Using newer version (^10.1.1), but this may cause build errors.
 WARN  conflicting versions for project dependency "typescript":

- ~4.2.4 injected by generator "@cnamts/vue-cli-plugin-vue-dash"
- ~4.1.5 injected by generator "@vue/cli-plugin-typescript"

Using newer version (~4.2.4), but this may cause build errors.
�  Installing additional dependencies...

npm ERR! code ETARGET
npm ERR! notarget No matching version found for eslint@^7.74.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'my-project'
npm ERR! notarget
```

**Comment by deraw** [comment]
Hello,

Thanks for opening this issue!

The ESLint version should be `^7.27.0`, we'll fix it in the next release.

In the meantime, you can fix it manually by setting the version in the `package.json` file.

**Comment by deraw** [comment]
Hello,

This issue has been resolved in the pull request [#1181](https://github.com/assurance-maladie-digital/design-system/pull/1181/files) on the [design-system](https://github.com/assurance-maladie-digital/design-system) repo and released in [v2.0.0-beta.11](https://github.com/assurance-maladie-digital/design-system/releases/tag/v2.0.0-beta.11)!
